aea/helpers/search/models.py

```diff
@@ -66,6 +66,12 @@ def __eq__(self, other):
             return False  # pragma: nocover
         return self.latitude == other.latitude and self.longitude == other.longitude
 
+    def __str__(self):
```

Comment:
to add?
```
def __repr__ (self):
   return str(self)
```
some times useful when print  a set of objects

Reply:
will do in separate pr